fulvio-almeida/1.i/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Sistemas de Informação</title>
+
+    <link type="text/css" href="style.css">
+</head>
+
+<body>
+
+    <h1>Sistemas de Informação</h1>
+
+    <p>O curso de Sistemas de Informação possui matriz curricular atual, com professores atuantes no mercado profissional e altamente conceituados e titulados. O curso capacita e proporciona oportunidades de carreira para os diplomados e que atendem plenamente
+        aos anseios do mercado, sempre exigente.</p>
+
+    <h2>Atuação Profissional</h2>
+    <p>O Profissional de Tecnologia da Informação tem a capacidade de compreender os problemas do mundo real, aplicação dos conhecimentos adquiridos na solução de problemas, capacidade de análise sistêmica dos problemas, capacidade de síntese e de realização
+        de projetos e, finalmente, capacidade de avaliação e julgamento. As habilidades pessoais desse profissional são: pensamento sistêmico, capacidade de resolução de problemas, pensamento crítico, disciplina pessoal, persistência, curiosidade, capacidade
+        de auto aprendizado e aberto a mudanças. Habilidades Interpessoais: trabalho colaborativo, capacidade de comunicação, capacidade para resolução conjunta de problemas. Conhecimentos Técnicos: abstração, representação e organização da informação,
+        uso de ferramentas computacionais para aplicação dos conhecimentos.</p>
+
+    <h3>Corpo Docente</h3>
```

Comment:
I can use the h2 because this is a specific heading, so I will change the h3, h4 when it necessary to use h2